src/Components/BookComponent.js

```diff
@@ -1,30 +1,35 @@
 import { PropTypes } from 'prop-types';
+import '../styles/Books.css';
 
 const Book = ({
   // eslint-disable-next-line react/prop-types
   cat, tit, aut, pro, cha, rmBook,
 }) => (
-  <div>
+  <div className="book-container">
     <div>
-      <p>{cat}</p>
-      <h2>{tit}</h2>
-      <p>{aut}</p>
-      <div>
-        <p>Comments</p>
-        <button type="button" onClick={rmBook}>Remove</button>
-        <p>Edit</p>
+      <p className="book-category">{cat}</p>
+      <h2 className="book-title">{tit}</h2>
+      <p className="author">{aut}</p>
+      <div className="book-buttons-container">
+        <p className="book-buttons">Comments</p>
+        <span className="Line-2" />
+        <button type="button" className="book-buttons" onClick={rmBook}>Remove</button>
+        <span className="Line-2" />
+        <p className="book-buttons">Edit</p>
       </div>
     </div>
-    <div>
+    <div className="progress">
+      <div className="Oval-2" />
       <div className="progress-circle">
-        <div>{pro}</div>
+        <div className="Percent-Complete">{pro}</div>
+        <div className="Completed">Completed</div>
       </div>
-      <div>Completed</div>
     </div>
+    <span className="separation-line" />
     <div>
-      <p>Current Chapter</p>
-      <p>{cha}</p>
-      <button type="button">Update Progress</button>
+      <p className="current-chapter">CURRENT CHAPTER</p>
+      <p className="chapter-number">{cha}</p>
+      <button type="button" className="update-progress">UPDATE PROGRESS</button>
     </div>
   </div>
 );
```

Comment:
- [ ] Hi @tkz96, great job so far! :+1: . Once more, let's again make sure that staying is EXACTLY the same as the one provided in Zeplin(Screenshot below). Font size, font-weight, color, and layouts should be EXACTLY the same as that in Zeplin.
- kindly include the user icon on your design.
- Make sure to adjust your items to take the entire weight of the `book-container` class
- The `remove` book button should be in line with `comments` and `edit `
- I have provided a screenshot of your design below with some annotations. Take a look at the annotations, compare them with the original design screenshot and kindly make the necessary adjustments. 
- **Screenshots:** 

Your Implementation | Zeplin Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/174445974-273044ec-d888-41a9-a652-594ccb3eb7e4.png) | ![](https://user-images.githubusercontent.com/63087491/174445993-2c340461-2a1b-49e4-80cf-5665f9d36096.png)